lib/ArduinoNative/Arduino.cpp

```diff
@@ -335,18 +322,36 @@ static int handleCommandLineArguments(PrgArguments& prgArguments, int argc, char
             /* fallthrough */
 
         default: /* Default */
-            printf("Usage: %s <option(s)>\nOptions:\n", programName);
-            printf("\t-h\t\t\tShow this help message.\n");          /* Help */
-            printf("\t-p <PORT NUMBER>\tSet SocketServer port.\n"); /* Port */
-            printf("\t-n <NAME>\t\tSet instace name.");             /* Name */
             status = -1;
             break;
         }
 
         option = getopt(argc, argv, availableOptions);
     }
 
+    /* Does the user need help? */
+    if (0 > status)
+    {
+        printf("Usage: %s <option(s)>\nOptions:\n", programName);
+        printf("\t-h\t\t\tShow this help message.\n");                /* Help */
+        printf("\t-n <NAME>\t\tSet robot name.\n");                   /* Robot Name */
+        printf("\t-p <PORT NUMBER>\tSet SocketServer port.\n");       /* SocketServer Port */

```

Comment:
Just in the line above shouldn't be a '\n', because the default value shall be behind, or?